extensions/ui/org.eclipse.smarthome.ui.basic/src/main/java/org/eclipse/smarthome/ui/basic/internal/render/ImageRenderer.java

```diff
@@ -94,4 +89,14 @@ public boolean canRender(Widget w) {
         sb.append(snippet);
         return null;
     }
+
+    private boolean isValidURL(String url) {
```

Comment:
That would be cool if you could move that method in the parent abstract class so that it could be used by any renderer class. I will need it in the video renderer class (take a look to my fix of this morning about MJPEG video).